nemo/core/callbacks.py

```diff
@@ -342,8 +342,8 @@ def on_step_end(self, state):
             if state["step"] % self._step_freq == 0 and self._step_freq > 0:
                 tensors_logged = {t: state["tensors"].get_tensor(t).cpu() for t in self._tensors_to_log}
                 # Always log learning rate
-                tensors_logged['LR'] = state["learning_rate"]
-                self._wandb_log(tensors_logged)
+                # tensors_logged['LR'] = state["learning_rate"]
+                self._wandb_log(tensors_logged, state["step"])
```

Comment:
👍  for removing/commenting LR, in most of our cases this is constant, so shouldn't be hardcoded in that callback

Reply:
No, this was a bug fix for an oversight on my part. I forgot to actually add `learning_rate` to the callback state.

Reply:
I have updated a bug fix for callbacks that incorporates the changes from this file: see #677